core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -529,8 +529,9 @@
       "When the gc runs it can make a lot of changes to the metadata, on completion, "
           + " to force the changes to be written to disk, the metadata and root tables can be flushed"
           + " and possibly compacted. Legal values are: compact - which both flushes and compacts the"
-          + " metadata; flush - which flushes only (compactions may be triggered if required); or none."
-          + " Since 2.0, the default is flush. Previously the default action was a full compaction."),
+          + " metadata; flush - which flushes only (compactions may be triggered if required); or none"),
+  GC_ENABLE_METRICS2("gc.enable.metrics2", "true", PropertyType.BOOLEAN,
```

Comment:
I think the following name change would make the property more consistent with similar props:

```suggestion
  GC_METRICS_ENABLED("gc.metrics.enabled", "true", PropertyType.BOOLEAN,
```